api/src/main/java/uk/gov/dhsc/htbhf/claimant/entitlement/CycleVoucherEntitlement.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.dhsc.htbhf.claimant.entitlement;
+
+import lombok.Data;
+
+import java.util.List;
+
+/**
+ * Represents the number (and value) of vouchers a claimant is entitled to for a given payment cycle.
+ */
+@Data
+public class CycleVoucherEntitlement {
```

Comment:
Let's rename to `PaymentCycleVoucherEntitlement`.

Reply:
Done.